typesense-java/src/main/java/org/typesense/model/Client.java

```diff
@@ -8,14 +8,34 @@
     private Api api;
 
     private Collections collections;
-
     private HashMap<String, Collection> individualCollections;
 
+    private Aliases aliases;
+    private HashMap<String, Alias> individualAliases;
+
+    private Keys keys;
+    private HashMap<String, Key> individualKeys;
+
+    public Health health;
+    public Operations operations;
+    public Metrics metrics;
+    public Debug debug;
+    public MultiSearch multiSearch;
+
     Client(Configuration configuration){
         this.configuration = configuration;
         this.api = new Api(configuration);
         collections = new Collections(api);
         this.individualCollections = new HashMap<>();
```

Comment:
Any reason this can't be `HashMap<String, Collection>`?

Reply:
Those parameters are declared while defining the variables like here 

[https://github.com/typesense/typesense-java/blob/project-restructure/typesense-java/src/main/java/org/typesense/model/Client.java#L11](url)

So while initialising the object Java automatically takes care of this. The same applies for all other similar class members